fcc-test-suite-template-master/index.html

```diff
@@ -60,6 +60,34 @@ <h1 id="title">Tito and Danny Bazar</h1>
          <button id="submit" type="submit" class="txts">Submit Feedback</button>
        </div>
      </form>
+     <ul class="projects">
+       <li class="project">
+         <img class="image" src="./img/snapshot.png" alt="">
```

Comment:
You did not add any description in alt text. Alt text text adds description to an image on a web page